### ISSUE TITLE
Could I have a compiled version of melonDS please?

### ISSUE THREAD

**Issue by CGG776** [mrdivdiz/melonDS-Vita-div#1]
I understand that it is possible to compile it myself but I never had any sucess, so could I please have a VPK? I am no good at doing these things myself even if there is an instruction manual or text there. Thanks in advance.

**Comment by mrdivdiz** [comment]
Yes, I compiled it so you can download 0.2 version in https://github.com/mrdivdiz/melonDS-Vita-div/releases/tag/0.2
Notice that melonDS is very slow.
